ansible/roles/k8s_docker/README.adoc

```diff
@@ -0,0 +1,9 @@
+== Role: Docker
+
+k8s_docker: Docker k8s configuration
+
+=== Description
+
+Docker configuration for Kubernetes.
```

Comment:
This is not correct as this role only enable the TCP connection with the docker daemon using the port 2376. This has nothing to do with k8s installation. It should not be named `k8s_docker` by the way ...

Reply:
renamed to `docker_config`